packages/components/src/components/ViewSettings.tsx

```diff
@@ -57,97 +145,115 @@ export const ViewSettings: React.FC<{
         </div>
       </HeadedSection>
 
-      <div className="pt-8">
-        <HeadedSection title="Distribution Display Settings">
-          <div className="space-y-2">
-            <Checkbox
-              register={register}
-              name="logX"
-              label="Show x scale logarithmically"
-              disabled={disableLogXSetting}
-              tooltip={
-                disableLogXSetting
-                  ? "Your distribution has mass lower than or equal to 0. Log only works on strictly positive values."
-                  : undefined
-              }
-            />
-            <Checkbox
-              register={register}
-              name="expY"
-              label="Show y scale exponentially"
-            />
-            <Checkbox
-              register={register}
-              name="distributionChartActions"
-              label="Show vega chart controls"
-            />
-            <Checkbox
-              register={register}
-              name="showSummary"
-              label="Show summary statistics"
-            />
-            <InputItem
-              name="minX"
-              type="number"
-              register={register}
-              label="Min X Value"
-            />
-            <InputItem
-              name="maxX"
-              type="number"
-              register={register}
-              label="Max X Value"
-            />
-            <InputItem
-              name="title"
-              type="text"
-              register={register}
-              label="Title"
-            />
-            <InputItem
-              name="tickFormat"
-              type="text"
-              register={register}
-              label="Tick Format"
-            />
-          </div>
-        </HeadedSection>
-      </div>
+      <DistributionViewSettings
```

Comment:
I should probably restore this component to what it was. The motivation for the split here is no longer neccesary.